jetty-http2/http2-common/src/main/java/org/eclipse/jetty/http2/HTTP2Cipher.java

```diff
@@ -19,311 +19,310 @@
+        "TLS_PSK_WITH_ARIA_256_CBC_SHA384",
+        "TLS_DHE_PSK_WITH_ARIA_128_CBC_SHA256",
+        "TLS_DHE_PSK_WITH_ARIA_256_CBC_SHA384",
+        "TLS_RSA_PSK_WITH_ARIA_128_CBC_SHA256",
+        "TLS_RSA_PSK_WITH_ARIA_256_CBC_SHA384",
+        "TLS_PSK_WITH_ARIA_128_GCM_SHA256",
+        "TLS_PSK_WITH_ARIA_256_GCM_SHA384",
+        "TLS_RSA_PSK_WITH_ARIA_128_GCM_SHA256",
+        "TLS_RSA_PSK_WITH_ARIA_256_GCM_SHA384",
+        "TLS_ECDHE_PSK_WITH_ARIA_128_CBC_SHA256",
+        "TLS_ECDHE_PSK_WITH_ARIA_256_CBC_SHA384",
+        "TLS_ECDHE_ECDSA_WITH_CAMELLIA_128_CBC_SHA256",
+        "TLS_ECDHE_ECDSA_WITH_CAMELLIA_256_CBC_SHA384",
+        "TLS_ECDH_ECDSA_WITH_CAMELLIA_128_CBC_SHA256",
+        "TLS_ECDH_ECDSA_WITH_CAMELLIA_256_CBC_SHA384",
+        "TLS_ECDHE_RSA_WITH_CAMELLIA_128_CBC_SHA256",
+        "TLS_ECDHE_RSA_WITH_CAMELLIA_256_CBC_SHA384",
+        "TLS_ECDH_RSA_WITH_CAMELLIA_128_CBC_SHA256",
+        "TLS_ECDH_RSA_WITH_CAMELLIA_256_CBC_SHA384",
+        "TLS_RSA_WITH_CAMELLIA_128_GCM_SHA256",
+        "TLS_RSA_WITH_CAMELLIA_256_GCM_SHA384",
+        "TLS_DH_RSA_WITH_CAMELLIA_128_GCM_SHA256",
+        "TLS_DH_RSA_WITH_CAMELLIA_256_GCM_SHA384",
+        "TLS_DH_DSS_WITH_CAMELLIA_128_GCM_SHA256",
+        "TLS_DH_DSS_WITH_CAMELLIA_256_GCM_SHA384",
+        "TLS_DH_anon_WITH_CAMELLIA_128_GCM_SHA256",
+        "TLS_DH_anon_WITH_CAMELLIA_256_GCM_SHA384",
+        "TLS_ECDH_ECDSA_WITH_CAMELLIA_128_GCM_SHA256",
+        "TLS_ECDH_ECDSA_WITH_CAMELLIA_256_GCM_SHA384",
+        "TLS_ECDH_RSA_WITH_CAMELLIA_128_GCM_SHA256",
+        "TLS_ECDH_RSA_WITH_CAMELLIA_256_GCM_SHA384",
+        "TLS_PSK_WITH_CAMELLIA_128_GCM_SHA256",
+        "TLS_PSK_WITH_CAMELLIA_256_GCM_SHA384",
+        "TLS_RSA_PSK_WITH_CAMELLIA_128_GCM_SHA256",
+        "TLS_RSA_PSK_WITH_CAMELLIA_256_GCM_SHA384",
+        "TLS_PSK_WITH_CAMELLIA_128_CBC_SHA256",
+        "TLS_PSK_WITH_CAMELLIA_256_CBC_SHA384",
+        "TLS_DHE_PSK_WITH_CAMELLIA_128_CBC_SHA256",
+        "TLS_DHE_PSK_WITH_CAMELLIA_256_CBC_SHA384",
+        "TLS_RSA_PSK_WITH_CAMELLIA_128_CBC_SHA256",
+        "TLS_RSA_PSK_WITH_CAMELLIA_256_CBC_SHA384",
+        "TLS_ECDHE_PSK_WITH_CAMELLIA_128_CBC_SHA256",
+        "TLS_ECDHE_PSK_WITH_CAMELLIA_256_CBC_SHA384",
+        "TLS_RSA_WITH_AES_128_CCM",
+        "TLS_RSA_WITH_AES_256_CCM",
+        "TLS_RSA_WITH_AES_128_CCM_8",
+        "TLS_RSA_WITH_AES_256_CCM_8",
+        "TLS_PSK_WITH_AES_128_CCM",
+        "TLS_PSK_WITH_AES_256_CCM",
+        "TLS_PSK_WITH_AES_128_CCM_8",
+        "TLS_PSK_WITH_AES_256_CCM_8"
+    ).map(String::toUpperCase).collect(Collectors.toSet());
 
     public static boolean isBlackListProtocol(String tlsProtocol)
     {
-        return __blackProtocols.get(tlsProtocol) != null;
+        return __blackProtocols.contains(tlsProtocol.toUpperCase());
     }
 
     public static boolean isBlackListCipher(String tlsCipher)
     {
-        return __blackCiphers.get(tlsCipher) != null;
+        return __blackCiphers.contains(tlsCipher.toUpperCase());
```

Comment:
This uses system default locale.
Change to `tlsProtocol.toUpperCase(Locale.US)`.
This will produce undesired side effects on on some, non-US, systems.